explorer/src/__tests__/entity/JobRun.test.ts

```diff
@@ -83,3 +84,26 @@ describe('fromString', () => {
     }
   })
 })
+
+describe('entity/jobRun/saveJobRunTree', () => {
+  it('overwrites taskRun values on conflict', async () => {
+    const [chainlinkNode, _] = await createChainlinkNode(
+      db,
+      'testOverwriteTaskRunsOnConflict'
+    )
+
+    const jr = fromString(JSON.stringify(fixture))
+    jr.chainlinkNodeId = chainlinkNode.id
+    await saveJobRunTree(db, jr)
+
+    const initial = await db.manager.findOne(JobRun)
+    expect(initial.taskRuns[0].confirmations).toEqual(0)
+
+    const updatedJr = fromString(JSON.stringify(updateFixture))
+    updatedJr.chainlinkNodeId = chainlinkNode.id
+    await saveJobRunTree(db, updatedJr)
+
+    const actual = await db.manager.findOne(JobRun)
+    expect(actual.taskRuns[0].confirmations).toEqual(3)
```

Comment:
Should this test all of the conflicting values?

Reply:
I don't think so. The fixture in use would need to be updated to have `transactionHash` and `transactionStatus`. There were no tests for on conflict at all, so this was added more for that coverage than specifically for `confirmations`. But it sounds like you believe it should test all conflict values.

Even if this had tests for previous onConflict columns, it would have been green, a false positive, since it doesn't fail on uncovered fields. Mundanely having the test match the columns doesn't seem very valuable to me.

Reply:
I don't mind being thorough though, and I'm indifferent. Give me the word and I'll have the test match the `onConflict` columns.

Reply:
Done.